.github/workflows/reusable-pr-is-ready.yaml

```diff
@@ -0,0 +1,91 @@
+name: PR Complete
+
+on:
+  workflow_call:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  All_checkboxes_checked:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - name: Fetch PR body and validate checkboxes
+        run: |
+          # 1. Fetch the pull request body via GitHub API
+          PR_BODY=$(curl -s \
+            -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }} \
+          | jq -r .body)
+
+          echo "Pull Request Body:"
+          echo "$PR_BODY"
+
+          # 2. Look for any unchecked boxes ("[ ]")
+          UNCHECKED=$(echo "$PR_BODY" | grep '\[ \]' || true)
+
+          # 3. If any are found, fail the build
+          if [ -n "$UNCHECKED" ]; then
+            echo "Error: Found unchecked checkboxes:"
+            echo "$UNCHECKED"
+            exit 1
+          else
+            echo "All checkboxes appear to be checked!"
+          fi
+  Docs_added_or_not_needed:
```

Comment:
Do we want to make these separate workflows, so we can more easily apply them separately?  They are very different checks, in my opinion, and github does not easily allow running different parts of one workflow file with different requirements

Reply:
* pr-check-boxes-complete.yaml
* proper-rst-documentation-edits.yaml
* proper-release-note-edits.yaml (this one already exists, create the rst documentation version with consistent naming?)

Reply:
Sure - works for me!

Reply:
Updated! :)